frontends/election-manager/src/components/smartcard_modal/smartcard_modal.test.tsx

```diff
@@ -641,15 +615,13 @@ test('Error handling', async () => {
     {
       cardData: makeAdminCard(electionHash),
       buttonToPress: 'Unprogram Card',
-      confirmButtonToPress: 'Yes',
-      expectedProgressText: 'Deleting all data on card',
+      expectedProgressText: 'Unprogramming Admin card',
```

Comment:
I find these sentences with just one uncapitalized word ("card") to look a bit weird. I would suggest capitalizing to "Admin Card" (it will also better match every other place we say "Admin Card" on the screen)

Reply:
https://github.com/votingworks/vxsuite/pull/2267/commits/df6babff8f28bf2527832277de026c74f4aa7f26